docs/user/contextplugin.md

```diff
@@ -0,0 +1,19 @@
+# FlowForge Persistent Context Plugin
+
+Projects running within FlowForge include a plugin that make it possible
+to store values in persistent context in a containered environment.
+
+
+### Known Limitations
+
+Currently, only async versions of context `get` and context `set` functions are supported. 
+View the readme for more info.
+
+
+### Requirements
+
+A persistent storage backend provided by a @flowforge/file-server instance
+
+### GitHub
```

Comment:
Why mention this for a user?

Reply:
I think it is relevant because it is required (wont work without it) in the containered env. These docs are not clear (to me) on who the audience is so it was written with a "nod" to requirements.  Without that line, an EE (or even CE) user may think his docker or k8s env should work? 

Open to suggestions.

Reply:
outdated